tsdef/twilio-video-tests.ts

```diff
@@ -30,6 +30,11 @@ function getDataTrack(track: Video.LocalDataTrack) {
   localDataTrack.send('hello world');
 }
 
+function makeDataTrack() {
```

Comment:
since this change involves `LocalDataTrackOptions`, this test should create this type on separate line:
perhaps something like
```
const localTrackOptions: LocalDataTrackOptions = { name: 'coolroom', logLevel: 'off' };
const localDataTrack: Video.LocalDataTrack = new Video.LocalDataTrack(localTrackOptions);
```